src/content/docs/guides/migrating-to-all-dataset.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Migrate queries to `all` dataset
+description: Assisting with query migration to the new dataset
+---
+
+New tables have been introduced in the HTTP Archive dataset, which are more efficient and easier to use. The `all` dataset contains all the data from the previous `pages`, `requests`, and other datasets. This guide will help you migrate your queries to the new dataset.
+
+## Migrating to `all.pages`
+
+### Page data schemas comparison
+
+previously | `all.pages`
+---|---
+date in a table name | [`date`](/reference/tables/pages/#date)
+client as `_TABLE_SUFFIX` | [`client`](/reference/tables/pages/#client)
+`url` in `pages.YYYY_MM_DD_client` | [`page`](/reference/tables/pages/#page)
+not available | [`is_root_page`](/reference/tables/pages/#is_root_page)
+not available | [`root_page`](/reference/tables/pages/#root_page)
+not available | [`rank`](/reference/tables/pages/#rank)
+`$.testID` within `payload` column in `pages.YYYY_MM_DD_client`, `wptid` column in `summary_pages.YYYY_MM_DD_client` | [`wptid`](/reference/tables/pages/#wptid)
+`payload`  in `pages.YYYY_MM_DD_client` | [`payload`](/reference/tables/pages/#payload)
+`req*`, `resp*` and other in `summary_pages.YYYY_MM_DD_client` | [`summary`](/reference/tables/pages/#summary)
+`$.CUSTOM_METRIC_NAME` within `payload` column in `pages.YYYY_MM_DD_client` | [`custom_metrics`](/reference/tables/pages/#custom_metrics)
+`report` in `lighthouse.YYYY_MM_DD_client` | [lighthouse](/reference/tables/pages/#lighthouse)
+`feature`, `type`, `id` in `blink_features.features` | `feature`, `type`, `id` in [`features`](/reference/tables/pages/#features)
+`category`, `app`, `info` in `technologies.YYYY_MM_DD_client` | `categories`, `technology`, `info` in [`technologies`](/reference/tables/pages/#technologies)
+not available | [`metadata`](/reference/tables/pages/#metadata)
+
+### Page query updates
+
+- From `blink_features.features` to `all.pages`.
+
+    ```diff lang="sql"
```

Comment:
Ah it looks like the rich formatting might have gotten lost in the downgrade :(

Alternatively, we could use the Tab components to show before/after queries

Reply:
Let's try tabs